test/index.test.js

```diff
@@ -499,6 +499,42 @@ describe('Optimizely', function() {
       });
     });
 
+    describe('#options.sendRevenueOnlyForOrderCompleted', function() {
+      beforeEach(function() {
+        analytics.stub(window.optimizely, 'push');
+      });
+
+      it('should send revenue only on Order Completed if onlySendRevenueOnOrderCompleted is enabled', function() {
```

Comment:
The default is false so this tests that `onlySendRevenueOnOrderCompleted` = false, thus when `properties.revenue` is present, it sends revenue